File: Java/10-Laço For/For.java
class For {
    public static void main(String args[]) {
        //Temos mais uma estrutra de repetição o for
        //A declaração condição de parada e incremento ou decremento
        //são feitos dentro da estrutura

        System.out.println("Contador de 0 a 10");
        for(int cont = 0; cont <= 10; cont++) {
            System.out.print(cont +" ");
        }

        System.out.println("\nContador de 10 a 0");
        for(int cont = 10; cont >= 0; cont--) {
            System.out.print(cont +" ");
        }
    }
}

File: PHP/10-Laço For/For.php
<?php

    //O for é outra estrutura de repetição dentor do php. Podemos declarar
    //a variavel contadora e fazer o incremento na declaração da própria estrutura

    for ($i = 1; $i <= 10; $i++) {//Incremento
        echo "$i ";
    }

    echo "<br/>";

    for ($i = 10; $i >= 1; $i--) {//Decremento
        echo "$i ";
    }

    echo "<br/>";

    for ($i = 0; $i <= 100; $i +=5){//Incremento 5 em 5
        echo "$i ";
    }

    echo "<br/>";

    for ($i = 20; $i >= 0; $i -= 2){//Decremento 2 em 2
        echo "$i ";
    }

?>